### PR TITLE
Replace Rotating Carousel with Modified Gregory Schier's Typing Carousel

### PR DESCRIPTION
**Pull request by cyntheticfox**
The implementation is largely as is under https://codepen.io/gschier/pen/jkivt and is licensed under an MIT license, so the license is included at the top of the javascript file.

The modification includes adding in a pull for `data-color` which is a JSON array of colors to show on the corresponding text.

Additionally, a smoothly-blinking block cursor replaces the static border-style cursor in Schier's implementation.

### CODE REVIEW

**Review by mmaekr — APPROVED** [review]
:+1:

File: index.html [hunk 1]
```diff
@@ -3,32 +3,26 @@
...
-                    </h2>
-                    <a href="/about" class="btn btn-default btn-contact wow fadeInUp" data-wow-delay=".9s">About Us</a>
-                    <!-- /slider -->
-                </div>
-            </div>
+  <div class="container">
+    <div class="row">
+      <div class="col-md-12 text-center">
+        <div class="block wow fadeInUp" data-wow-delay=".3s">
+          <section class="cd-intro">
```
Comment (lilyinstarlight) [inline index.html]: I'm pretty sure `cd-intro` is no longer necessary as nothing defines it now (well actually it wasn't in slider.css before but all of the `cd-` ones seemed to be there).

```suggestion
          <section>
```

**Review by lilyinstarlight — CHANGES_REQUESTED** [review]
Remove ` cd-headline slide` from `h1` class in 404.html since you removed the CSS file for it (and it wasn't doing anything anyway) and remove the `.cd-headline.slide .cd-words-wrapper` selector from main.css since this just sets a color that used to get overridden (so did nothing before) and now does even less given that no longer matches anything.

Thanks @houstdav000!

File: index.html (hunk 1 above)
Comment (cyntheticfox) [reply index.html]: Might as well get rid of that `<section>` tag altogether since there's no CSS bound to it then.

**Review by lilyinstarlight — APPROVED** [review]

**Pull request merged**